src/hooks/useGeolocationHandler.ts

```diff
@@ -3,7 +3,8 @@ import { executeSearch } from '../utils/search-operations';
 import { getUserLocation } from '../utils/location-operations';
 import { useCallback, useState } from 'react';
 
-const LOCATION_FIELD_ID = 'builtin.location';
+const GEOLOCATION_FIELD_ID = 'builtin.location';
+const LOCATION_FIELD_IDS = [GEOLOCATION_FIELD_ID, 'builtin.region', 'address.countryCode'];
```

Comment:
can these consts be use in filtersearch?

Reply:
I didn't use them in the FilterSearch PR because that was off of the release branch, which doesn't have Geolocation. similarly, develop doesn't have those FilterSearch changes, so I can't directly update it in this PR either. given the current naming of the consts (i.e. `GEOLOCATION_FIELD_ID`), I think it might be better to wait until the FilterSearch changes are in develop and there's more context for geolocation. but, if you feel strongly about it, I can extract these consts into a common file and make another PR to refer to them from there in the release branch

Reply:
that's alright, not a big deal